recipe/meta.yaml

```diff
@@ -61,8 +69,7 @@ outputs:
         - zlib
       run:
         - ucrt  # [win]
-        - openssl
-        - zlib
+        - zlib  # [win]
```

Comment:
Strictly speaking, this is now only necessary for the pkgconfig test of `libgrpc` (which wants to find `zlib.pc` for the requires_private); we can either live with the extra run-req here or disable the pkgconfig test (or patch the pkgconfig metadata, but as it's already marked private, I'm not 100% sure we want to do that).